src/User.ts

```diff
@@ -12,6 +12,27 @@ export interface UserProfile {
     azp?: string;
     at_hash?: string;
     auth_time?: number;
+
+    // Other standard claims - https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims
+    name?: string;
+    given_name?: string;
+    family_name?: string;
+    middle_name?: string;
+    nickname?: string;
+    preferred_username?: string;
+    profile?: string;
+    picture?: string;
+    website?: string;
+    email?: string;
+    email_verified?: boolean;
+    gender?: string;
+    birthdate?: string;
+    zoneinfo?: string;
+    locale?: string;
+    phone_number?: string;
+    phone_number_verified?: boolean;
+    address?: Record<string, unknown>; // https://openid.net/specs/openid-connect-core-1_0.html#AddressClaim
```

Comment:
I'd again leverage TSDoc here:

```suggestion
    /** @see https://openid.net/specs/openid-connect-core-1_0.html#AddressClaim */
    address?: Record<string, unknown>
```

Doing it this way allows for it to be picked up by the language server and in the Typedoc!

![image](https://user-images.githubusercontent.com/4993980/144939919-24304734-756c-4e5a-9f2a-d696bad063ab.png)